honeybadger-io/honeybadger-js
# feat(webpack): rely on plugin-core

## Status
**READY**

## Description
Closes https://github.com/honeybadger-io/honeybadger-js/issues/1013

## Related PRs
https://github.com/honeybadger-io/honeybadger-js/pull/1193
https://github.com/honeybadger-io/honeybadger-js/pull/1195

## Todos
- [x] Manual testing of multiple sourcemap file upload
- [x] Modify the example project to have sourcemaps actually get applied to an error -- this would make us more confident that the upload worked correctly
- [x] Manual testing of behavior when sourcemap upload fails 

## Steps to Test or Reproduce
In `/webpack`
```
npm run build
```

In `/webpack/example`
```
npm run build
```
You should see happy logging. 
<img width="647" alt="Screen Shot 2023-10-13 at 4 08 10 PM" src="https://github.com/honeybadger-io/honeybadger-js/assets/16493493/750b04db-5558-4a2b-b4ff-7c640478e577">


You should see the deployment and sourcemap file in the Honeybadger UI.
